libs/ledger-live-common/src/families/cosmos/api/Cosmos.ts

```diff
@@ -270,4 +270,20 @@ export class CosmosAPI {
 
     return patchOperationWithHash(operation, data.tx_response.txhash);
   };
+
+  simulate = async (tx_bytes: number[]): Promise<BigNumber> => {
```

Comment:
Maybe a comment to explain what's simulated + what's returned

Reply:
Done